.gitignore

```diff
@@ -1,5 +1,3 @@
 .Rproj.user
 inst/doc
-.RHistory
-
-Workflow_*
```

Comment:
I think we need to remove this from being saved on github. This is not code right?

Reply:
Yep